### DISCUSSION TITLE
Enhancement: Add option to check for videos/images in Archive.org/Wayback Machine

### DISCUSSION THREAD

**Discussion by metapea** [TumblThreeApp/TumblThree#510] [Ideas]
With the Wayback Machine, you can check old twitter and tumblr posts and not just the main page also tweets and individual posts with put a `*` at the end of the main blog/profile or at the "_URLs_" page.
There can be a lot of twitter posts in the Wayback Machine, but currently there's no gui to download them all. 
So adding support to add blogs with urls like `https://web.archive.org/web/*/(url of profile)/*` for all of the archived posts or `https://web.archive.org/web/20240000000000*/(url of profile)/` for just the archives of the front page, would be nice.
Alternately, you can just make it into a check box instead, and have an option to download into it's own folder or the same one.
Just note that with tumblr, the Wayback Machine sometime gets confuse on tumblr's older urls like `https://(user).tumblr.com/` and can auto redirects to the new url `https://www.tumblr.com/(user)`. Also there's other web archive sties like archive.today as well. 
Edit: Also discovered in tumblr some of files can be still in it's servers (most likely from others rebloging), add a check box for finding urls like, `https://web.archive.org/web/(date)/http://(number).media.tumblr.com/(id)/tumblr_(id)_500.jpg` into `http://(number).media.tumblr.com/(id)/tumblr_(id)_1280.jpg` to check if the file is still on tumblr and then download it.

**Comment by thomas694** [comment]
Currently there is no plan to integrate this site.
If enough people are interested in it and someone likes to implement, we welcome it.
For now moving it to the ideas section.
